frontend/public/components/daemon-set.jsx

```diff
@@ -23,7 +23,7 @@ import {
   Selector,
 } from './utils';
 
-export const menuActions = [Kebab.factory.EditEnvironment, ...Kebab.factory.common];
+export const menuActions = [Kebab.factory.AddStorage, Kebab.factory.EditEnvironment, ...Kebab.factory.common];
```

Comment:
Not new but, I while ago we had a discussion about constructing the `menuActions` and we align with @spadgett that we should destruct the `Kebab.factory` for actions and use those, similarly as you do below in *frontend/public/components/stateful-set.jsx*

Reply:
I think this is OK since it's a copy of the array. My concern is passing around the original `Kebab.factory.common` array in case it could ever accidentally be mutated.

Reply:
OK, let's defer to that :+1: